src/app/api/routes/media.py

```diff
@@ -8,26 +8,44 @@
 router = APIRouter()
 
 
-@router.post("/", response_model=MediaOut, status_code=201)
+@router.post("/", response_model=MediaOut, status_code=201, summary="Create a media related to a specific device")
 async def create_media(payload: MediaIn):
+    """
+    Creates a media related to specific device, based on device_id as argument
+
+    Below, click on "Schema" for more detailed information about arguments
+    or "Example Value" to get a concrete idea of arguments
+    """
     return await crud.create_entry(media, payload)
 
 
-@router.get("/{media_id}/", response_model=MediaOut)
+@router.get("/{media_id}/", response_model=MediaOut, summary="Get information about a specific media")
 async def get_media(media_id: int = Path(..., gt=0)):
+    """
+    Based on a media_id, retrieves information about the given media
+    """
     return await crud.get_entry(media, media_id)
 
 
-@router.get("/", response_model=List[MediaOut])
+@router.get("/", response_model=List[MediaOut], summary="Get the list of all medias")
 async def fetch_media():
+    """
+    Retrieves the list of all medias with each related information
+    """
```

Comment:
nitpicking here, but media comes from latin and is actually already plural. In English, the plural is media (and some use medium as singular). But no need for the "s" :)

Reply:
Oops indeed ! Thanks !